packages/directions-finder/src/direction-buttons.tsx

```diff
@@ -52,6 +64,21 @@ function DirectionButtons({
             {t(['hyeonjieseo-gilmudgi', '현지에서 길묻기'])}
           </Button>
         ) : null}
+
+        {hasLineBreak ? <LinkBreak /> : null}
+
+        {isGrabSupported ? (
```

Comment:
```suggestion
        {onCallGrabButtonClick ? (
```

부가적인 기능은 이벤트 핸들러 유무로 판단하는 곳이 많아서 그렇게 맞추는 것이 좋을 것 같습니다.

Reply:
6cb3611f14f2c100749f7185dd6f2f681d448bb6 여기서 수정했습니다!